api/src/main/java/org/openmrs/module/registrationcore/api/RegistrationCoreService.java

```diff
@@ -131,22 +144,52 @@
 	 */
 	List<String> findSimilarFamilyNames(String searchPhrase);
 
+	/**
+	 * Query to MPI server to find patient with Id "patientId".
```

Comment:
Wouldn't it be clearer to replace patientId with identifier?

Reply:
changed